Generation - json empty lines update

vince-roll20 opened the pull request:
- removed extra lines at the end of the json TBH, this file was showing changes in my local fork. Just trying to wrap up any merge conflicts.

## Changes / Comments






## Roll20 Requests

Comments are very helpful for reviewing the code changes. Please answer the relevant questions below in your comment.

- [ ] Does the pull request title have the sheet name(s)? Include each sheet name.
- [ ] Is this a bug fix?
- [ ] Does this add functional enhancements (new features or extending existing features) ?
- [ ] Does this add or change functional aesthetics (such as layout or color scheme) ? 
- [ ] If changing or removing attributes, what steps have you taken, if any, to preserve player data ?
- [ ] If this is a new sheet, did you follow [Building Character Sheets standards](https://wiki.roll20.net/Building_Character_Sheets#Roll20_Character_Sheets_Repository) ?

If you do not know English. Please leave a comment in your native language.

roll20deploy commented:
[Character Sheet Info](https://app.roll20.net/managesheets/Generation/roll20-character-sheets) *Roll20 Internal Use only.*

mperes commented:
Hey there @vince-roll20 , your pull request has some conflicts that need to be resolved before I merge it. Please rebase it against master and fix those. Thanks for all the work you do!

vince-roll20 commented:
OK.  I rebased the branch and added "legacy": true in order to match what was merged yesterday https://github.com/Roll20/roll20-character-sheets/blame/master/Generation/sheet.json by Wing-it.
I'm still not sure why this file is causing issues for me given I did not make any changes to it.  Maybe my local fork is corrupted...?  IDK
